src/page-objects/storefront/Account.ts

```diff
@@ -18,6 +19,7 @@ export class Account implements PageObject {
         this.shippingAddressCardTitle = page.getByRole('heading', { name: 'Default shipping address' });
         this.newsletterCheckbox = page.getByLabel('Yes, I would like to');
         this.newsletterRegistrationSuccessMessage = page.getByText('You have successfully subscribed to the newsletter.');
+        this.customerGroupRequestMessage = page.getByText(/Access to customer group ".*" requested\./);
```

Comment:
I would get this locator dynamically if possible if we want to check the entire text, [see here](https://github.com/shopware/acceptance-test-suite/commit/f079431044175f416ca1457d0a9fda91f79d2d27#diff-6df0c5e7cb97654b2850e125bbfb6e9e8804c9b8e41a5715cd79d91beed917e1R88) wdyt?

Reply:
I applied a helper method.